README.md

```diff
@@ -40,13 +40,7 @@
 Add [@else] to your build tool:
 
 ```bash
-npm install postcss-at-else --save-dev
-```
-
-#### Node
-
-```js
-require('postcss-at-else').process(YOUR_CSS, { /* options */ });
```

Comment:
Not a 100% sure.
But I don't think `require('a-plugin').process(...)` still works with postcss 8.0.0

This would require adding `postcss` as a dependency again and adding `module.exports.process ...`

Reply:
Good catch! 👍 Thank you!